packages/untp-test-suite/src/interfaces/utils/common.ts

```diff
@@ -9,4 +18,44 @@ export async function readJsonFile<T>(filePath: string): Promise<T | null> {
   } catch (error) {
     return null;
   }
-}
\ No newline at end of file
+}
+
+export const getTemplateName = (testSuiteResult: IValidatedCredentials): string => {
+  if (testSuiteResult.errors === null) {
+    return TemplateName.pass;
+  }
+
+  if (Array.isArray(testSuiteResult.errors) && testSuiteResult.errors.length > 0) {
+    const getErrors = testSuiteResult.errors as ErrorObject[];
```

Comment:
I think variables that represent objects or entities should typically be named with nouns, while functions and methods should use verbs or action-oriented names.